app/views/home/tos_faq.html.erb

```diff
@@ -420,7 +422,7 @@ Please use our search functions for this rather than creating a separate work.</
 			<p>Unless there's an independent violation of the Content Policy, we won't intervene in collection decisions, even if they are arbitrary, biased, or wrong. You may want to add tags to your fanwork that will be of interest to people who are fans of relevant collections.
 			</p>
 			<h5 class="heading">How can I start a collection?</h5>
-			<p><a href="https://archiveofourown.org/archive_faqs/25">Please consult our Tutorials.</a></p>
+			<p><a href="https://archiveofourown.org/archive_faqs/25">Please consult our Tutorials.</a> Please note that participants may provide information to the maintainer for purposes of participating in a collection or challenge. Any use of this information other than to manage the collection or challenge is a violation of our Terms of Service and can result in the termination of the maintainer’s account.</p>
```

Comment:
Also tiny annoying nitpick: "maintainer’s account" has a smart/curly apostrophe when we use straight '